imageroot/events/user-domain-changed/20configure_ldap

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+import json
+import sys
+import agent
+import os
+
+event = json.load(sys.stdin)
+
+if event.get('domain') != os.getenv('LDAP_DOMAIN'):
+    exit(0)
+
+if 'node_id' in event and str(event['node_id']) != os.getenv('NODE_ID'):
+    exit(0) # ignore event if the source is not in our node
```

Comment:
We should always restart, despite the nod_id: the account provider could be everywhere.
```suggestion
```